libraries/c_sdk/standard/https/src/iot_https_client.c

```diff
@@ -232,10 +232,10 @@ static int _httpParserOnBodyCallback( http_parser * pHttpParser,
                                       size_t length );
 
 /**
- * @brief Callback from http-parser to indicate it reached the end of the HTTP response messsage.
+ * @brief Callback from http-parser to indicate it reached the end of the HTTP response message.
  *
  * The end of the message is signalled in a HTTP response message by another "\r\n" after the final header line, with no
- * entity body; or it is singalled by "\r\n" at the end of the entity body.
+ * entity body; or it is singaled by "\r\n" at the end of the entity body.
```

Comment:
signalled

Reply:
signaled since 1776

Reply:
That's cool. But I don't think that's what it says.

Reply:
oh oops thanks